src/pages/ProfilesExplorerView/infrastructure/series/SeriesDataSource.ts

```diff
@@ -34,10 +32,10 @@ export class SeriesDataSource extends RuntimeDataSource {
       state: LoadingState.Done,
       data: [
         {
-          name: 'Services',
+          name: 'PyroscopeSeries',
           fields: [
             {
-              name: 'Service',
+              name: 'PyroscopeSerie',
```

Comment:
nit: the singular of series is ...series.

![english](https://github.com/user-attachments/assets/ff27751a-61f9-41c8-acad-0e0bbfc20c87)

Reply:
Damn.